src/ServiceControl/MessageFailures/Handlers/UnArchiveMessagesHandler.cs

```diff
@@ -41,6 +41,7 @@ await session.SaveChangesAsync()
 
             await domainEvents.Raise(new FailedMessagesUnArchived
             {
+                DocumentIds = failedMessages.Select(x => x.UniqueMessageId).ToArray(),
```

Comment:
We may want to batch these to keep the message sizes low. I am not sure if there are natural limits to how big this collection can be.

Reply:
What kind of batch size do you think we should do here?
Is there a metric we can use?